aviary/examples/external_subsystems/engine_NPSS/engine_variables.py

```diff
@@ -0,0 +1,28 @@
+import aviary.api as av
+
+AviaryAircraft = av.Aircraft
+AviaryDynamic = av.Dynamic
+
+
+# ---------------------------
+# Aircraft data hierarchy
+# ---------------------------
+
+class Aircraft(AviaryAircraft):
+	
+	class Engine(AviaryAircraft.Engine):
+		DESIGN_MACH = "aircraft:engine:design_mach"
+		DESIGN_ALTITUDE = "aircraft:engine:design_alt"
+		DESIGN_MASS_FLOW = "aircraft:engine:design_mass_flow"
+		DESIGN_NET_THRUST = "aircraft:engine:design_net_thrust"  # will need to use this to calculate the Aviary-core variable aircraft.Propulsion.TOTAL_SCALED_SLS_THRUST using number of engines
+
+
+# ---------------------------
+# Dynamics data hierarchy
+# ---------------------------
+
+class Dynamic(AviaryDynamic):
+	
+	class Engine:
+		ELECTRIC_SHAFT_POWER = "dynamic:engine:electric_shaft_power"
```

Comment:
This should have an entry in the engine_variable_meta_data

Reply:
Added,  uploaded change